bip-0085.mediawiki

```diff
@@ -360,36 +360,58 @@ OUTPUT
 * DERIVED ENTROPY=f7cfe56f63dca2490f65fcbf9ee63dcd85d18f751b6b5e1c1b8733af6459c904a75e82b4a22efff9b9e69de2144b293aa8714319a054b6cb55826a8e51425209
 * DERIVED PWD=_s`{TW89)i4`
 
-===RSA===
+===GPG Keys===
 
-Application number: 828365'
+Application number is dependant on the key type.
 
-The derivation path format is: <code>m/83696968'/828365'/{key_bits}'/{key_index}'</code>
+{|
+!OpenGPG Key Type
+!Application Number
+|-
+| RSA
+| 828365'
+|-
+| ECC(Curve25519)
+| 828366'
+|-
+| ECC(secp256k1)
+| 828367'
+|-
+| ECC(NIST)
+| 828368'
+|-
+| ECC(Brainpool)
+| 828369'
+|-
+|}
 
-The RSA key generator should use BIP85-DRNG as the input RNG function.
+The RSA key generator should use BIP85-DRNG as the input RNG function. Likewise, any ECC key types over 256bit should use BIP85-DRNG.
 
-===RSA GPG===
+====Primary Keys and Subkeys====
 
-Keys allocated for RSA-GPG purposes use the following scheme:
+Keys allocated for GPG purposes use the following scheme:
 
- - Main key <code>m/83696968'/828365'/{key_bits}'/{key_index}'</code>
- - Sub keys:  <code>m/83696968'/828365'/{key_bits}'/{key_index}'/{sub_key}'</code>
+ - Primary key <code>m/83696968'/{key_type}'/{key_bits}'/{key_index}'</code>
```

Comment:
Here and elsewhere these list nested code blocks don't render as expected (... > View file)